src/zenml/zen_stores/rest_zen_store.py

```diff
@@ -2836,6 +2836,17 @@ def delete_model_version_artifact_link(
             route=f"{MODEL_VERSIONS}/{model_version_id}{ARTIFACTS}",
         )
 
+    def delete_all_model_version_artifact_link(
+        self,
+        model_version_id: UUID,
+    ) -> None:
+        """Deletes all model version to artifact links.
```

Comment:
```suggestion
        """Deletes all links between model version and an artifact.
```

Or something like that?